group_project_v2/stage.py

```diff
@@ -631,7 +632,7 @@ def load_peer_feedback(self, request, suffix=''):  # pylint: disable=unused-argu
         return webob.response.Response(body=json.dumps(results))
 
     def do_submit_review(self, submissions):
-        peer_id = submissions["review_subject_id"]
+        peer_id = int(submissions["review_subject_id"])
```

Comment:
@Kelketek actually, I'm a bit wrong - there aren't any catch all clause - I'll update this one as well.